app/src/main/java/com/dankideacentral/dic/TwitterUtil.java

```diff
@@ -19,11 +20,23 @@
     private static TwitterUtil instance = new TwitterUtil();
 
     public static void init(Context applicationContext) {
+        // Get twitter auth token & secret from shared preferences
+        SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(applicationContext);
+
+        String twitterAuthToken = preferences.getString(
+                applicationContext.getString(R.string.twitter_auth_preference), null);
+        String twitterAuthTokenSecret = preferences.getString(
+                applicationContext.getString(R.string.twitter_auth_secret_preference), null);
+
+        // Create twitter authentication configuration
         ConfigurationBuilder configurationBuilder = new ConfigurationBuilder();
-        configurationBuilder.setOAuthConsumerKey(applicationContext.getString(R.string.twitter_consumer_key));
-        configurationBuilder.setOAuthConsumerSecret(applicationContext.getString(R.string.twitter_consumer_secret));
-        Configuration configuration = configurationBuilder.build();
-        twitterFactory = new TwitterFactory(configuration);
+        configurationBuilder.setDebugEnabled(true)
+                .setOAuthConsumerKey(applicationContext.getString(R.string.twitter_consumer_key))
+                .setOAuthConsumerSecret(applicationContext.getString(R.string.twitter_consumer_secret))
+                .setOAuthAccessToken(twitterAuthToken)
+                .setOAuthAccessTokenSecret(twitterAuthTokenSecret);
+
+        twitterFactory = new TwitterFactory(configurationBuilder.build());
         twitter = twitterFactory.getInstance();
```

Comment:
pass the access token in when calling getInstance.   twitterFactory.getInstance(accessToken) .. should solve the problem. It was required for the TwitterStreamService